### PR TITLE
Fixes #3131 - Output information at the end of the foreman-installer

### PR DESCRIPTION
**Pull request by GregSutcliffe**
Produces output like:

```
# foreman-installer
  Success!
    You can now login at https://test.domain.org
    A proxy is running at https://test.domain.org:8443
    A puppetmaster has been set up on port 8140
  There is a full log /var/log/foreman-installer/foreman-installer.log
```

or on error:

```
# foreman-installer
  Something went wrong! Check the log below for ERROR level lines
  There is a full log /var/log/foreman-installer/foreman-installer.log
```

### CODE REVIEW

**Comment by ekohl** [comment]
Looks good, but maybe you can check if authentication is enabled and mention default username + password?

File: bin/foreman-installer [hunk 1]
```diff
@@ -1,6 +1,38 @@
...
+require 'yaml'
 require 'kafo'
 
-CONFIG_FILE = "config/foreman-installer.yaml"
-KafoConfigure.run
+CONFIG_FILE = "/usr/share/foreman-installer/config/foreman-installer.yaml"
+result = KafoConfigure.run
+exit 0 if result.nil? # --help invocation
+
+answers = YAML.load_file(result.config.app[:answer_file])
```
Comment (ares) [inline bin/foreman-installer]: Why don't you use parameter values that are already parsed? Something like

``` ruby
if result.config['foreman_proxy']
```

reloading and reparsing answer file seems wrong, also becase params can have some extra parsing logic

File: bin/foreman-installer [hunk 2]
```diff
@@ -1,6 +1,38 @@
...
+
+# Puppet status codes say 0 for unchanged, 2 for changed succesfully
+unless [0,2].include? result.exit_code
+  puts "  Success!"
+
+  # Foreman UI?
+  if ( answers.include? 'foreman' and answers['foreman'] != false)
```
Comment (ares) [inline bin/foreman-installer]: Could you please change `and` for `&&`? `and` and `or` are usually used for flow control, `&&` and `||` are used in boolean expressions

**Comment by GregSutcliffe** [comment]
@ekohl done

@ares Updated to use https://github.com/theforeman/kafo/issues/15

**Comment by ares** [comment]
Kafo 0.0.16 released. This versions contains patch from https://github.com/theforeman/kafo/pull/15

**Comment by GregSutcliffe** [comment]
Tested with kafo 0.0.16, works as expected (on Wheezy)

File: bin/foreman-installer [hunk 3]
```diff
@@ -1,6 +1,56 @@
...
+  if module_enabled? 'foreman_proxy'
+    say "  * <%= color('Foreman Proxy', :info) %> is running at <%= color('#{get_param('foreman_proxy','registered_proxy_url')}', :info) %>"
+  end
+
+  # Puppetmaster?
+  if ( module_enabled?('puppet') && ( get_param('puppet','server') != false ) )
+    say "  * <%= color('Puppetmaster', :info) %> is running at <%= color('port #{get_param('puppet','server_port')}', :info) %>"
+  end
+else
+  say "  <%= color('Something went wrong!', :bad) %> Check the log for ERROR-level output"
```
Comment (lzap) [inline bin/foreman-installer:49]: Minor suggestion: Can we add which log file to check? We will be doing this million times, I'd love just to copy and paste it into grep command... :-)

Nice thanks!

Comment (GregSutcliffe) [inline bin/foreman-installer:49]: It does, see line 54

**Comment by GregSutcliffe** [comment]
bump, are we including this in rc3?

**Comment by domcleal** [comment]
Looks OK to me.  Can you add highline to the Gemfile as we now explicitly require it?

**Comment by lzap** [comment]
Merging as there is no Gemfile in the repo :-) Thanks guys

**Pull request merged**